### PR TITLE
[MIG] base_import_async: Migration to 13.0

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
LG

File: base_import_async/models/base_import_import.py [hunk 1]
```diff
@@ -32,11 +33,12 @@
 class BaseImportImport(models.TransientModel):
     _inherit = "base_import.import"
 
-    @api.multi
-    def do(self, fields, options, dryrun=False):
+    def do(self, fields, columns, options, dryrun=False):
         if dryrun or not options.get(OPT_USE_QUEUE):
             # normal import
-            return super(BaseImportImport, self).do(fields, options, dryrun=dryrun)
+            return super(BaseImportImport, self).do(
```
Comment (sebalix) [inline base_import_async/models/base_import_import.py]:
```suggestion
            return super().do(
```

File: base_import_async/models/base_import_import.py [hunk 2]
```diff
@@ -70,42 +72,39 @@ def do(self, fields, options, dryrun=False):
...
-        encoding = options.get(OPT_ENCODING, "utf-8")
+        encoding = options.get(OPT_ENCODING) or "utf-8"
         writer.writerow(fields)
         for row in data:
             writer.writerow(row)
         # create attachment
         datas = base64.encodebytes(f.getvalue().encode(encoding))
         attachment = self.env["ir.attachment"].create(
-            {"name": file_name, "datas": datas, "datas_fname": file_name}
+            {"name": file_name, "datas": datas, "store_fname": file_name}
```
Comment (sebalix) [inline base_import_async/models/base_import_import.py]: I'm not sure we should update `store_fname` field which handled internally by the filestore, maybe `name` replaces `datas_fname` now?
```suggestion
            {"name": file_name, "datas": datas}
```

**Comment by sebalix** [comment]
@guewen black missing on the updated files :)

**Comment by guewen** [comment]
> @guewen black missing on the updated files :)

What, github suggestions do not run black? :stuck_out_tongue_winking_eye:
Fixed the last commit.

**Review by sebalix — APPROVED** [review]

**Pull request merged**